jinahub/rankers/SimpleRanker/simpleranker.py

```diff
@@ -21,12 +23,15 @@ class MinRanker(Executor):
     def __init__(
         self,
         metric: str = 'cosine',
+        ranking: str = 'min',
```

Comment:
this name is not so clear to me. But I'm not sure what could be a better naming. maybe ranking_function, aggregrate_function..

Reply:
I personally prefer ranking over the longer renking_function. I will write a few more comments so it is more clear.